modules/030-cloud-provider-yandex/docs/FAQ_RU.md

```diff
@@ -60,3 +60,13 @@ reserved: true
    ```shell
    kubectl edit mc cloud-provider-yandex
    ```
+
+## Добавление CloudStatic-узлов в кластер
+
+К виртуальным машинам, которые вы хотите добавить к кластеру в качестве узлов, добавьте в метаданные (Изменить ВМ -> Метадата) ключ `node-network-cidr` со значением равному установленному в кластере `nodeNetworkCIDR`.
```

Comment:
```suggestion
В метаданные виртуальных машин, которые вы хотите включить в кластер в качестве узлов, добавьте (Изменить ВМ -> Метадата) ключ `node-network-cidr` со значением `nodeNetworkCIDR` для кластера.
```

Reply:
Did some optional rephrase. Imho, it looks better now. Feel free to apply it as you see fit